tools/abel/validators/README.md

```diff
@@ -0,0 +1,192 @@
+# ABEL Spreadsheet
+
+### Overview
+The ABEL Spreadsheet serves as a user interface for the ABEL application. The
+spreadsheet is the user-friendly interface that allows users to create a
+[Building Configuration file](../../../ontology/docs/building_config.md) from scratch.
+
+## Google Sheets template
+
+The format of an ABEL spreadsheet is well-defined, and it is suggested that you use the following [Google Sheets Template](https://docs.google.com/spreadsheets/d/1tcLjFnHiXUT-xh5C1hRKiUVaUH_CzgSI8zFQ_B8q7vs/copy#gid=980240783)
+
+Below is a definition of each table and column in the ABEL spreadsheet:
+- [Site](#site)
+- [Entities](#entities)
+- [Entity Fields](#entity-fields)
+- [States](#states)
+- [Connections](#connections)
+
+### Generating GUIDs
+
+There are two use cases for ABEL, brown field and green field. Currently ABEL
+only supports Green Field buildings or Building Configs under the [Initialize
+operation](../../../ontology/docs/building_config.md#initialize)
+
+#### Green Field
```

Comment:
same.

can you link to an external page what does greenfield and brownfield means?